shell/app/locales/en.json

```diff
@@ -1180,6 +1180,7 @@
     "Supported Android package versions": "Supported Android package versions",
     "Supported iOS package versions": "Supported iOS package versions",
     "Sydney": "Sydney",
+    "Set tags of {ip}": "Set tags of {ip}",
```

Comment:
Don’t add and edit local files manually, use erda-cli to generate it automatically